packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss

```diff
@@ -31,10 +32,18 @@
 
 	.body {
 		flex: 1 1 auto;
+		padding: 0 15px;
+		padding-bottom: $padding-larger;
```

Comment:
Can you see with the UX if this is right ? I see it in guideline but the guideline spelcifies a space between the text and the bottom instead of between the day button and the bottom. And this space seems very big in practice.